nymea-app/configuredhostsmodel.cpp

```diff
@@ -99,12 +101,55 @@ void ConfiguredHostsModel::removeHost(int index)
         qCWarning(dcApplication()) << "Cannot remove connection at index" << index;
         return;
     }
+
     beginRemoveRows(QModelIndex(), index, index);
-    m_list.takeAt(index)->deleteLater();
+
+    ConfiguredHost *host = m_list.takeAt(index);
+    qCDebug(dcApplication()) << "Remove configured host" << host->name() << host->uuid().toString();
+
+    QSettings settings;
+
+    QString hostUuidString = host->uuid().toString();
+
+    qCDebug(dcApplication()) << "-> Remove stored token";
+    settings.beginGroup("jsonTokens");
+    if (settings.contains(hostUuidString)) {
+        settings.remove(hostUuidString);
+    }
+    settings.endGroup();
+
+    qCDebug(dcApplication()) << "-> Remove cached hosts";
+    settings.beginGroup("HostCache");
+    settings.beginGroup(hostUuidString);
+    settings.remove("");
```

Comment:
Removing "nothing"?

Reply:
One way of QSettingts to remove the entire group and all chrildren

Reply:
Ahh nice to know.